sugarscape.py

```diff
@@ -34,11 +34,13 @@ def __init__(self, configuration):
                                     "universalSugarIncomeInterval": configuration["environmentUniversalSugarIncomeInterval"],
                                     "sugarscapeSeed": configuration["seed"]}
         self.seed = configuration["seed"]
+        self.debug = configuration["debugMode"]
         self.environment = environment.Environment(configuration["environmentHeight"], configuration["environmentWidth"], self, environmentConfiguration)
         self.environmentHeight = configuration["environmentHeight"]
         self.environmentWidth = configuration["environmentWidth"]
-        self.configureEnvironment(configuration["environmentMaxSugar"], configuration["environmentMaxSpice"])
-        self.debug = configuration["debugMode"]
+        # Maintain backwards compatibility for configs without adjacencyMode
+        adjacencyMode = configuration.get("adjacencyMode", None)
```

Comment:
`get()` sets `adjacencyMode` to whatever it is in the config settings, or to a default value of `None` if there's no `adjacencyMode` in the config settings.